website/content/docs/concepts/filtering/worker-tags.mdx

```diff
@@ -137,3 +133,24 @@ Following are some examples of using these values in filters:
   know that you have only one value, an equivalent would be `"/tags/region/0" == "us-east-1"`.
 
 - Grouping: `("us-east-1" in "/tags/region" and "/name" == "web-prod-us-east-1") or "webservers" in "/tags/type"`
+
+# Target Worker Filtering
+
+Once workers have tags, it is possible to use these tags to control which
+workers are allowed to handle a given session by specifying a `worker_filter`
```

Comment:
```suggestion
manage a given session by specifying a `worker_filter`
```

Reply:
Feel free to modify my suggestions above if it changes the intent of that sentence.